cmd/installer/cmd/root.go

```diff
@@ -44,4 +44,5 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&options.Upgrade, "upgrade", false, "Indicates that the install is being performed by an upgrade")
 	rootCmd.PersistentFlags().BoolVar(&options.Force, "force", false, "Indicates that the install should forcefully format the partition")
 	rootCmd.PersistentFlags().BoolVar(&options.Zero, "zero", false, "Indicates that the install should write zeros to the disk before installing")
+	rootCmd.PersistentFlags().StringVar(&options.PreviousVersion, "previous-version", "", "The previous version of Talos")
```

Comment:
Torn on what to call this flag. From the perspective of the thing running this container (i.e. Talos) it is more accurately called "current-version".

Reply:
We also refer to the "current" entry as a "fallback." So we could have `current-version` or `fallback-version`.